src/main/java/seedu/duke/Parser.java

```diff
@@ -42,30 +36,56 @@ public static boolean parseUserInput(String userInput) {
         switch (inputCommand) {
         case "create":
             try {
-                executeCreate(userInputSplit[1]);
-            } catch (IndexOutOfBoundsException e) {
+                assert inputParams != null;
+                executeCreate(inputParams);
+            } catch (IndexOutOfBoundsException | NullPointerException e) {
                 Ui.printCreateFormatError();
             }
             break;
 
         case "edit":
-            executeEdit(userInputSplit[1]);
+            try {
+                assert inputParams != null;
+                executeEdit(inputParams);
+            } catch (NumberFormatException | IndexOutOfBoundsException | NullPointerException e) {
+                Ui.printUnknownTripIndexError();
+            }
             break;
 
         case "open":
-            executeOpen(userInputSplit[1]);
+            try {
+                assert inputParams != null;
+                executeOpen(inputParams);
+            } catch (NumberFormatException | IndexOutOfBoundsException e) {
+                Ui.printSingleUnknownTripIndexError();
+                System.out.println();
```

Comment:
Just checking, is this supposed to print something or isit just a new line?

Reply:
I just copied and pasted the original, I think maybe it isnt supposed to be there(?)